src/v3/src/util/makeRequest.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import fetch from 'cross-fetch';
+
+type RequestOptions = {
+  url: string;
+  timeout: number;
+  method: 'GET' | 'POST';
+  data?: string;
+};
+
+/**
+* Temporary request client can be removed if this functionality is added
+* to auth-js library.
+* @see https://oktainc.atlassian.net/browse/OKTA-561852
```

Comment:
This was moved from [LoopbackProbe.tsx](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/components/LoopbackProbe/LoopbackProbe.tsx#L30-L76), considering deleting this comment since the ticket referenced is resolved as _won't do_, but will await feedback on this.